admin-dev/themes/new-theme/js/components/entity-search-input.ts

```diff
@@ -243,6 +244,12 @@ export default class EntitySearchInput {
 
       const $entity = $(event.target).closest(this.options.entityItemSelector);
 
+      if (!this.options.removeModal) {
```

Comment:
Well i don't believe that this component is suitable for this case then as its the same if I spam the modal everytime you want to select something from choice list (because your previous selection is getting deleted, beaware!) :smile: But ok i bring it back and we can improve in any other pr's if necessary.